examples/plugins/src/package-json/src/integration/types.ts

```diff
@@ -0,0 +1,16 @@
+export type DependencyTypes =
```

Comment:
Union types should have a singular name, since variables will have only one value (`depType: DependencyType` is more natural than `depType: DependencyTypes`).

```suggestion
export type DependencyType =
```